### ISSUE TITLE
"BREAKING CHANGE: " keyword does not trigger major version bump

### ISSUE THREAD

**Issue by limonte** [semantic-release/commit-analyzer#108]
### Steps to reproduce:

```sh
git clone https://github.com/sweetalert2/sweetalert2.git
npm i -g semantic-release@16.0.0-beta.16 @semantic-release/changelog@3 @semantic-release/exec@3 @semantic-release/git@7;
semantic-release
```

### Output:

```
[2:49:40 PM] [semantic-release] » i  Running semantic-release version 16.0.0-beta.16
[2:49:41 PM] [semantic-release] » √  Loaded plugin "verifyConditions" from "@semantic-release/changelog"
[2:49:41 PM] [semantic-release] » √  Loaded plugin "verifyConditions" from "@semantic-release/npm"
[2:49:41 PM] [semantic-release] » √  Loaded plugin "verifyConditions" from "@semantic-release/github"
[2:49:41 PM] [semantic-release] » √  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
[2:49:41 PM] [semantic-release] » √  Loaded plugin "generateNotes" from "@semantic-release/release-notes-generator"
[2:49:41 PM] [semantic-release] » √  Loaded plugin "prepare" from "@semantic-release/exec"
[2:49:41 PM] [semantic-release] » √  Loaded plugin "prepare" from "@semantic-release/changelog"
[2:49:41 PM] [semantic-release] » √  Loaded plugin "prepare" from "@semantic-release/npm"
[2:49:42 PM] [semantic-release] » √  Loaded plugin "prepare" from "@semantic-release/git"
[2:49:42 PM] [semantic-release] » √  Loaded plugin "publish" from "@semantic-release/npm"
[2:49:42 PM] [semantic-release] » √  Loaded plugin "publish" from "@semantic-release/github"
[2:49:42 PM] [semantic-release] » √  Loaded plugin "addChannel" from "@semantic-release/npm"
[2:49:42 PM] [semantic-release] » √  Loaded plugin "addChannel" from "@semantic-release/github"
[2:49:42 PM] [semantic-release] » √  Loaded plugin "success" from "@semantic-release/github"
[2:49:42 PM] [semantic-release] » √  Loaded plugin "success" from "@semantic-release/exec"
[2:49:42 PM] [semantic-release] » √  Loaded plugin "fail" from "@semantic-release/github"
[2:51:03 PM] [semantic-release] » ‼  Run automated release from branch master in dry-run mode
[2:51:11 PM] [semantic-release] » √  Allowed to push to the Git repository
[2:51:11 PM] [semantic-release] » i  Start step "verifyConditions" of plugin "@semantic-release/changelog"
[2:51:11 PM] [semantic-release] » √  Completed step "verifyConditions" of plugin "@semantic-release/changelog"
[2:51:11 PM] [semantic-release] » i  Start step "verifyConditions" of plugin "@semantic-release/npm"
[2:51:11 PM] [semantic-release] [@semantic-release/npm] » i  Verify authentication for registry https://registry.npmjs.org/
[2:51:13 PM] [semantic-release] » √  Completed step "verifyConditions" of plugin "@semantic-release/npm"
[2:51:13 PM] [semantic-release] » i  Start step "verifyConditions" of plugin "@semantic-release/github"
[2:51:13 PM] [semantic-release] [@semantic-release/github] » i  Verify GitHub authentication
[2:51:14 PM] [semantic-release] » √  Completed step "verifyConditions" of plugin "@semantic-release/github"
[2:51:14 PM] [semantic-release] » i  Found git tag v7.33.1 associated with version 7.33.1 on branch master
[2:51:14 PM] [semantic-release] » i  Found 26 commits since last release
[2:51:14 PM] [semantic-release] » i  Start step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: drop Bower support (#1377)

semantic-release from master branch, dist branch was needed to support Bower
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore(deps): drop detect-browser dependency (#1374)

We are detecting IE11 only, no need for
the additional dependency for that simple task.
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore: remove withGlobalDefaults enhancer
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: remove withNoNewKeyword enhancer (#1372)

From now on the recommended way to use SweetAlert2 is:

Swal.fire({...options})
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore: import/export .js files explicitly

Needed for ES modules
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: remove swal.noop()
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore: remove legacy adaptInputValidator (#1371)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: rename $swal2-validationerror -> $swal2-validation-message (#1370)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: remove getButtonsWrapper()
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: remove setDefault and resetDefaults (#1365)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: remove extraParams (#1363)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: remove showValidationError and resetValidationError (#1367)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore(travis): decrease max bundlesize 16KB -> 15KB
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: remove useRejections and expectRejections (#1362)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: dismissReason: overlay -> backdrop (#1360)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: BREAKING CHANGE: drop Android 4.4 support (#1359)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore(package): update detect-browser to version 4.0.1 (#1358)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: NEW MAJOR START: disable automated releasing
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore: add 'toast', 'popup', 'accessible' keywords
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore: add Jeff Keith to Supporters
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore: bump dev-deps
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: Update rollup to version 1.0.0 � (#1353)

* chore(package): update rollup to version 1.0.0

* chore(package): update lockfile yarn.lock
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore: custom classes default value: null -> empty string (#1352)
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore: optimize resumeTimer test
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore(saucelabs): use 'Android GoogleAPI Emulator'

The latest version for 'Android Emulator' is outdated ATM,
it's 6.0. For Android GoogleAPI Emulator it's 7.1
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analyzing commit: chore(release): 7.33.1 [skip ci]

## [7.33.1](https://github.com/sweetalert2/sweetalert2/compare/v7.33.0...v7.33.1) (2018-12-22)

### Bug Fixes

* **d.ts:** add customContainerClass definition ([#1351](https://github.com/sweetalert2/sweetalert2/issues/1351)) ([c5f11e7](https://github.com/sweetalert2/sweetalert2/commit/c5f11e7))
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  The commit should not trigger a release
[2:51:14 PM] [semantic-release] [@semantic-release/commit-analyzer] » i  Analysis of 26 commits complete: no release
[2:51:14 PM] [semantic-release] » √  Completed step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
[2:51:14 PM] [semantic-release] » i  There are no relevant changes, so no new version is released.
```

### Actual behavior:

> There are no relevant changes, so no new version is released

### Expected behavior:

The new major should be released, because there are multiple commits with `BREAKING CHANGE: ` keyword in it

**Comment by pvdlg** [comment]
Please see the [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).
`BREAKING CHANGE:` must be in the footer of the commit. You wrote it in the subject.

**Comment by abdel-ships-it** [comment]
@pvdlg I had this problem as well, ~~I think the readme [here](https://github.com/semantic-release/semantic-release#commit-message-format) example should perhaps be changed~~

Never mind the example in the readme is perfect, I misread that and thought the commit has to start with `BREAKING CHANGE`

**Comment by EvanCarroll** [comment]
What a horrible error message...

**Comment by gr2m** [comment]
Hi Evan,

you clearly are dissatisfied with semantic release: https://twitter.com/TheEvanCarroll/status/1441447068397015044

We'd ask you at this point to refrain from venting your frustration within the semantic-release community and fork the project instead.

**Comment by EvanCarroll** [comment]
It's not venting. It makes sense to follow the Angular Commit message format, but this series of errors is not very informative,

```text
ℹ  Analyzing commit: BREAKING CHANGE: Changes from slice_into_subrepo
ℹ  The commit should not trigger a release
```

It would be much more useful to say,

```text
ℹ  Analyzing commit: detected "BREAKING CHANGE:" but it's not in the commit message footer, per Angular's "Commit Message Format"
ℹ  The commit should not trigger a release
```

I do have a fork of most of the semantic-release/* stuff that cleans up the errors, but not commit-analyzer yet. I'm rewriting the project in Rust, but I'm no where near a release. [I also give back online on StackOverflow to try to eliminate the confusion without patching.](https://stackoverflow.com/q/70061964/124486)
